### DISCUSSION TITLE
Angular Boilerplate Admin-Template

### DISCUSSION THREAD

**Discussion by sourcefactory** [vmware/clarity#6556] [Q&A]
I am excited about Clarity!
Is there a boilerplate admin template (admin) for Angular with the UI Clarity...?

**Comment by gnomeontherun** [comment]
Hi @sourcefactory we don't have any boilerplates specifically built, but there are some community based themes that use Clarity for various platforms you might be able to find (not sure how well maintained they are and we don't maintain a list).
